src/MSBuild/LiveLogger/LiveLogger.cs

```diff
@@ -42,9 +42,20 @@ internal record NodeStatus(string Project, string? TargetFramework, string Targe
     {
         public override string ToString()
         {
+            string duration = Stopwatch.Elapsed.TotalSeconds.ToString("F1");
```

Comment:
Is the double -> string formatting locale-aware if done in code like this? My suggestion is to omit the `ToString` call here and change the placeholder in `Strings.resx` to something like `{0:F1}`.

Reply:
Yes `double.ToString` does respect `Thread.CurrentThread.CurrentCulture`  - I have also tested it. 
Have though about if formatting rules belongs to resx. In this case I proffered it in application logic as I viewed presented decimal point precision as UX design responsibility which do not, IMO, belongs to translation team. 
If we choose to change it in future, such change would not trigger need for translation.

Reply:
Thank you. I guess I could argue either way - for the app logic to own the formatting so it's easier to change in the future and for the resource files to contain it so it can be localized. FWIW, we already have a precedence where the format is specified in .resx and it's translated to the same format in all languages. So it probably doesn't matter.

https://github.com/dotnet/msbuild/blob/c6e6cd447ac3944b351247d995f32b4d2010d409/src/Build/Resources/Strings.resx#L1481